_posts/2020-02-29-f12.markdown

```diff
@@ -51,7 +51,7 @@ sudo losetup -d "${LOOP_DEVICE}"
 These commands have some disadvantages.
 At first they require superuser privileges to mount the images.
 This should not be necessary for such a simple process and may not work in
-containerized environments.
+containerised  environments.
```

Comment:
```suggestion
containerized  environments.
```

Ich glaube, das war schon vorher richtig so

* https://hackernoon.com/what-is-containerization-83ae53a709a6
* https://www.ibm.com/cloud/learn/containerization

Reply:
Gut was neues dazu gelernt